ServiceApp/src/org/droidplanner/services/android/core/drone/autopilot/apm/ArduPilot.java

```diff
@@ -198,6 +196,11 @@ public boolean executeAsyncAction(Action action, final ICommandListener listener
                 CommonApiUtils.startMission(this, forceModeChange, forceArm, listener);
                 return true;
 
+            case MissionActions.ACTION_CHANGE_MISSION_SPEED:
```

Comment:
@chaviw move this logic to the `GenericMavLinkDrone` class. The `DO_CHANGE_SPEED` command is part of the common xml definition file, so other autopilot types should support it as well.